spec/features/people/edit_person_details_spec.rb

```diff
@@ -135,8 +135,7 @@
   end
 
   it "includes a select field to change the day of an openingtime" do
-    @openingtime = create :openingtime
-    @person.openingtimes = [@openingtime]
+    @openingtime = FactoryBot.create(:openingtime, timeable: @person)
```

Comment:
Why did you change this? Unrelated to locations?

Reply:
At some point we had modified the factory for openingtimes and probably changed this as well.
Since the factorybot is back to the state it was before, we will revert this as well (there is no reason for this change now).